leshan-core/src/main/java/org/eclipse/leshan/core/request/RegisterRequest.java

```diff
@@ -34,6 +35,7 @@
     private final Long lifetime;
     private final String lwVersion;
```

Comment:
I should maybe change the type of this attribute to `Version` (same in `Registration` object) :thinking: